keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -929,6 +998,10 @@ Major milestones might include:
 - when the KEP was retired or superseded
 -->
 
+  - 1.26, KEP created.
+  - 1.26, alpha implementation.
+  - 1.27, beta implementation.
```

Comment:
This usually is in the form of:
- date: action.

Reply:
Done